docs/guidance/EnterpriseScaleAnalytics-SelfService.md

```diff
@@ -0,0 +1,117 @@
+# Scaling and Self-Service in Enterprise-Scale Anayltics
+
+Efficient scaling within an Enterprise Data Platform is a much-desired goal of many organizations and IT organizations as business units should be enabled to build their own (data) solutions and applications that fulfill their requirements and needs. However. achieving this goal is a serious challenge as many existing data platforms are not built around the core concepts of scalability and decentralized ownership. This is not only true from an architectural standpoint, but also becomes noticeable when looking at the team structure within many existing data platforms.
```

Comment:
Missing comma (,) after "**However**" 

Consider rephrasing the sentence " This is not only true from an architectural standpoint, but also becomes noticeable when looking at the team structure within many existing data platforms." 

Rephrase to ... 

"This is not only true from architectural standpoint but also applicable to incubment operations model which underpins existing data platforms."